### DISCUSSION TITLE
DayPickerInput

### DISCUSSION THREAD

**Discussion by usmansagri** [gpbl/react-day-picker#1900] [Support]
Hi All,

Good day!

My question maybe related to outdated feature of Day picker 

I wanted to know if we can add captionElement to DayPickerInput to drop down month and year? 

I am unable to find any documentation. Kindly help if you have any idea.

**Comment by gpbl** [comment]
@usmansagri https://react-day-picker-v7.netlify.app/examples/input-state has some examples using the v7 `DayPickerInput`. Does this documentation helps you?
